algviz/graphs/graphs.py

```diff
@@ -0,0 +1,36 @@
+import pygraphviz as pgv
+
+
+def nodesToLocations(nodes, edges, algo="dot"):
+    """ Maps node to their coordinates.
+    """
+    G = pgv.AGraph()
+    G.graph_attr['fixedsize'] = True
+    for node in nodes:
+        G.add_node(node, fixedsize=True, width=node.width, height=node.height)
+    for (node1, node2) in edges:
+        G.add_edge(node1, node2)
+    G.layout(prog=algo)
+    dictNodesLocations = {}
+    for node in G.nodes():
+        dictNodesLocations[node.get_name()] = node.attr['pos']
+    return dictNodesLocations
+
+
+class Node:
+    def __init__(self, name, width, height):
+        self.name = name
+        self.width = width
+        self.height = height
+
+    def __str__(self):
+        return self.name
+
+
+nodeNames = ['a', 'b', 'c']
```

Comment:
You should move this test to a separate test file, and expand to include more tests. Specifically, we should have tests to make sure that more complicated graphs are visualized well: i.e. graphs with cycles, graphs with self-loops, graphs with different size nodes, and graphs with nodes of irregular shape (at least).